civihr_default_theme/scss/components/_header.scss

```diff
@@ -173,14 +172,16 @@
 
     &:last-child {
       > a {
-        border: none;
+        border: 0;
       }
     }
   }
 
-  @media(min-width: $screen-md-max) {
-    display: flex;
+  .menu-link-hr-admin {
+    order: 9999;
```

Comment:
Please add a comment why 9999, something like `to ensure it appears the last in the list`